File: lei_lds2122_grupo2-main/Backend/Data/ViewModels/Response/ErrorsVM.cs
using Backend.Data.ViewModels.Types;

namespace Backend.Data.ViewModels.Response
{
    /// <summary>
    /// View model class for the list of errors response
    /// </summary>
    public class ErrorsVM
    {
        public List<ErrorType> Errors { get; set; }

        /// <summary>
        /// Class' constructor method
        /// </summary>
        /// <param name="errors">List with the errors</param>
        public ErrorsVM(List<ErrorType> errors)
        {
            this.Errors = errors;
        }


    }
}

File: lei_lds2122_grupo2-main/Frontend/src/app/viewModels/responses/ErrorsVM.ts
import { ErrorType } from "../Types/ErrorType";

/**
 * View Model for a response with errors
 */
export class ErrorsVM {

    /**
     * Construct's an instance of {@link ErrorsVM}
     * 
     * @param errors errors array
     */
    constructor(
        public errors: Array<ErrorType>
    ) { }
}